src/commands/spawn.h

```diff
@@ -1,472 +1,76 @@
-        "v029_car_smallmodernsedan_civilian_01",
-        "v029_car_smallmodernsedan_civilian_02",
-        "v030_car_oldmini_civilian_01",
-        "v030_car_oldmini_civilian_02",
-        "v031_car_racinghothatch_civilian_01",
-        "v031_car_racinghothatch_encounter_takedown",
-        "v032_car_luxurysportssedan_civilian_01",
-        "v033_car_modernmini_civilian_01",
-        "v033_car_modernmini_civilian_02",
-        "v034_car_oldtruck_commercial_01",
-        "v034_car_oldtruck_commercial_cargo_barrel",
-        "v034_car_oldtruck_commercial_cargo_barrel_mission_special",
-        "v034_car_oldtruck_commerical_cargo_random",
-        "v034_car_oldtruck_commerical_cargo_sargento_intro",
-        "v035_car_modernvan_civic_ambulance_01",
-        "v035_car_modernvan_commerical_01",
-        "v035_car_modernvan_garland",
-        "v036_car_modernpickup_civilian_01",
-        "v036_car_modernpickup_civilian_02",
-        "v037_car_modernbus_commerical_01",
-        "v038_car_modernsuv_civilian_01",
-        "v038_car_modernsuv_civilian_02",
-        "v039_car_oldcampervan_civilian_01",
-        "v039_car_oldcampervan_civilian_02",
-        "v040_car_oldcompact_civilian_01",
-        "v040_car_oldcompact_sargento",
-        // "v041_car_",
-        "v042_car_racingsedan_civilian",
-        "v042_car_racingsedan_racing_01",
-        // "v043_car_",
-        // "v044_car_",
-        "v045_car_minetruck_commercial_01",
-        "v046_car_racingbuggy_civilian",
-        "v046_car_racingbuggy_military",
-        "v046_car_racingbuggy_racing_01",
-        // "v047_car_",
-        // "v048_car_",
-        // "v049_car_",
-        "v050_car_toyjeep_civilian",
+    struct PropertyFileEntry {
+        uint32_t hash;
+        uint32_t data;
+        uint8_t  type;
+    };
 
-        // boats
-        "v100_boat_fanboat_civilian",
-        "v101_boat_smalljet_military",
-        "v101_boat_smalljet_rebel",
-        "v102_boat_heavypatrol_lightning",
-        "v102_boat_heavypatrol_military_01",
-        "v102_boat_heavypatrol_rebel_01",
-        "v103_boat_corvette_military_01",
-        "v103_boat_corvette_rebel_01",
-        "v104_boat_landingtransport_commercial",
-        "v104_boat_landingtransport_commercial_lightningrod_escort",
-        "v104_boat_landingtransport_commercial_signal_jammer",
-        "v104_boat_landingtransport_rebel",
-        "v105_boat_sailboat_civilian_01",
-        "v106_boat_motoryacht_bomb_special",
-        "v106_boat_motoryacht_civilian_01",
-        "v107_boat_racingboat_bomb_special",
-        "v107_boat_racingboat_civilian_01",
-        "v108_boat_largeoldfishing_civilian_01",
-        "v108_boat_largeoldfishing_civilian_01_garland_intro",
-        "v109_boat_ferry_commercial",
-        "v109_boat_ferry_commercial_garlandintro",
-        "v110_boat_jetski_civilian_01",
+    struct PropertyFileResult {
+        uint8_t *          base;
+        PropertyFileEntry *entry;
+        uint8_t            error;
+    };
```

Comment:
```suggestion
    };
#pragma pack(pop)
```
Closing for above `push`.